### ISSUE TITLE
ZSH Parse Error after upgrading to 0.4.3 

### ISSUE THREAD

**Issue by sandangel** [ajeetdsouza/zoxide#95]
I got this error after upgrading to 0.4.3

```sh
(eval):25: defining function based on alias `zi'
(eval):25: parse error near `()'
```

**Comment by ajeetdsouza** [comment]
Is your issue related to https://github.com/ajeetdsouza/zoxide/issues/94?

The change was that zoxide now defines the `zi` function regardless of `--no-aliases`, which I'm reconsidering now due to the clashes with zinit. I'll remove this in the next update and add a more human-friendly error, but for now, do look at the linked issue for a quickfix.

**Comment by sandangel** [comment]
@ajeetdsouza I guess so. Thanks for pointing me to the issue.

**Comment by ajeetdsouza** [comment]
Btw, this is fixed in v0.5.0.
